### ISSUE TITLE
PyDMRelatedDisplay - Bring to Front when New Window already exists

### ISSUE THREAD

**Issue by hhslepicka** [slaclab/pydm#666]
**Describe the bug**
When using the Related Display button and the `Open in New Window` option is checked, a new window is always created when clicking the button.

**Expected behavior**
When clicking the button, if the window still exists, bring it to the front.

**Comment by mattgibbs** [comment]
I disagree with this being a bug.  I *want* the ability to open up multiple instances of the same display, and use it frequently.

--Matt

**Comment by hhslepicka** [comment]
Why would you open multiple times the same screen with the absolute same information?

**Comment by klauer** [comment]
Could it be an option or a dialog that pops up asking if you _really_ want a new window when one is already there?

**Comment by hhslepicka** [comment]
I am okay with adding an option (`bringExistingToFront`)  or some other name that makes sense... and default it to False to keep existing behavior.

**Comment by slactasha** [comment]
Adding to this old issue - I would like to have the option. In EDM it's the 'Dups Allowed' checkbox that is unchecked by default. 

Like Matt, I often open multiple instances of the same display, especially when it's a template that I use for many devices and I want to display some of them side by side.
